cosine_similarity_model.py

```diff
@@ -0,0 +1,141 @@
+import joblib
+import logging
+import numpy as np
+from pymongo import MongoClient
+import pandas as pd
+from sklearn.metrics.pairwise import cosine_similarity
+from sklearn.feature_extraction.text import CountVectorizer
+
+
+class MovieRecommender:
+    def __init__(self):
+        self.logger = self.setup_logger()
+        self.all_movie_ids = []
+        self.all_movie_titles = {}
+        self.selected_movies = pd.DataFrame()
+
+
+    @staticmethod
+    def setup_logger():
+        logger = logging.getLogger('MovieRecommender')
+        logger.setLevel(logging.INFO)
+        handler = logging.StreamHandler()
+        formatter = logging.Formatter('%(asctime)s - %(levelname)s - %(message)s')
+        handler.setFormatter(formatter)
+        logger.addHandler(handler)
+        return logger
+
+
+    @staticmethod
```

Comment:
why does this method need to be static?

Reply:
If the method does not need access to self or cls, it can be static. Isn't it?